lint.swift

```diff
@@ -0,0 +1,547 @@
+#!/usr/local/bin/swift-sh
+import AnyLint // @Flinesoft ~> 0.8.2
```

Comment:
Codacy found an issue: [Header comments should be consistent with project patterns.](https://app.codacy.com/gh/Flinesoft/HandySwift/pullRequest?prid=5730214)